src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.InvokeNativeHandler.cs

```diff
@@ -156,5 +153,36 @@ private static HttpMessageHandler CreateNativeHandler()
 
             return (HttpMessageHandler)_nativeHandlerMethod!.Invoke(null, null)!;
         }
+
+        private object InvokeNativeHandlerGetter(Func<MethodInfo> getMethod, [CallerMemberName] string? cachingKey = null)
```

Comment:
Perhaps we could use CallerLineNumberAttribute or some other key which is not a next string in the assembly